#! /usr/bin/env python
PAKAGE = "learning_parameter"

from dynamic_reconfigure.parameter_generator_catkin import *

if __name__ == "__main__":
  gen = ParameterGenerator()

  gen.add("double_param", double_t, 0, "max_distance^2", default=4, min=0.01, max=100)

  exit(gen.generate(PAKAGE, "learning_parameter", "try_dyparam"))
